### ISSUE TITLE
class-properties syntax error

### ISSUE THREAD

**Issue by atticoos** [nickzuber/storybook-parser-utils#1]
`acorn` parser hits some trouble if it the contents contain syntaxes that require custom parsing

eg
```js
class MyStory extends React.Component {
  state = {...} // Syntax error here
  ...
}

storiesOf('Story', module)
  .add('My Story', () => <MyStory />)
```

**Comment by nickzuber** [comment]
Good find — another great thing about `acorn` is its extendability. I think we should be able to fix this easily with another acorn library.

**Comment by nickzuber** [comment]
Fixed in b192e30026fcb578678de32dad0a8cfe10a61ca9

**Comment by nickzuber** [comment]
Fixed in b192e30026fcb578678de32dad0a8cfe10a61ca9
